VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.spec.ts

```diff
@@ -32,19 +37,29 @@ describe('UnreadMessagesComponent', () => {
     beforeEach(() => {
         const unreadMessages = conference.participants.map(
             p =>
-                new UnreadAdminMessageResponse({
-                    number_of_unread_messages: 5,
-                    participant_id: p.id
-                })
+                new UnreadAdminMessageModel(
+                    new UnreadAdminMessageResponse({
+                        number_of_unread_messages: 5,
```

Comment:
I know these tests were here before but any chance we could have a const that defines this number since it's used multiple places.

Reply:
Done